09. cli-and-http-request/clihelper/WriteUsingOutputLoggers.go

```diff
@@ -0,0 +1,8 @@
+package clihelper
+
+func WriteUsingOutputLoggers(c CliBindingProperties, d []byte) {
+	_, err := w.Writer(c.WritersCollection.WriterConfiguration.WriteToFilelocation, d)
```

Comment:
![image](https://user-images.githubusercontent.com/4561204/91750943-c007db80-ebe5-11ea-9cf4-2eb4eca824c9.png)
Same as this one, we have to process.

Reply:
done